parent/pom.xml

```diff
@@ -164,6 +164,11 @@
         <version>1.0</version>
         <scope>provided</scope>
       </dependency>
+      <dependency>
+        <groupId>org.hibernate.javax.persistence</groupId>
+        <artifactId>hibernate-jpa-2.0-api</artifactId>
+        <version>1.0.1.Final</version>
```

Comment:
It would be keeping with the pattern if we move the version in a property defined in the `properties` section above.

Reply:
Not sure this is really a pattern we have, it's rather 50:50 when you look at all the dependencies. If we need the version as a property, we can still change that in the future.